src/actions/AccountReferralActions.js

```diff
@@ -195,18 +196,27 @@ async function validatePromoCards(cards: MessageTweak[]): Promise<MessageTweak[]
   } catch (e) {}
 
   const out = []
+  let hasLinkedBankBool
```

Comment:
Not super important but hasLinkedBankBool and notLinkedBank appear to be wyre specific so maybe the variable names should reflect that clarity in the code and for the individual creating the promos

Reply:
The request was for a generic check on bank account linkage that could be across any provider. However I'd agree that the promos are likely more specific to a provider. I'd prefer not changing the name, but instead specifying a  providerId/pluginId list to look for. I'll make the change.